xlsx.js

```diff
@@ -7680,6 +7680,12 @@ function write_ws_xml_cols(ws, cols) {
   o[o.length] = "</cols>";
   return o.join("");
 }
+function write_ws_xml_sheetviews(ws, opts, idx, wb) {
+	var sview = ({workbookViewId:"0"});
+	// $FlowIgnore
+	if((((wb||{}).Workbook||{}).Views||[])[0]) sview.rightToLeft = wb.Workbook.Views[0].RTL ? "1" : "0";
```

Comment:
One-liners are the style of this project: but that is wrong and dangerous and hard to read. I am not a project owner, but this is something to consider.

Reply:
Can you explain in your pull (now 2 years stale) what this does, exactly?